src/pages/SellerUpload/ImageUploader.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import { useState } from 'react';
+import axios from 'axios';
+
+import styled from 'styled-components';
```

Comment:
- 굳이 띄어주실 필요가 없어보입니다!
```suggestion
import React from 'react';
import { useState } from 'react';
import axios from 'axios';
import styled from 'styled-components';
```

Reply:
수정했습니다!! 감사힙니다